src/__tests__/jest/actions/socialLoginAction.test.js

```diff
@@ -0,0 +1,46 @@
+import moxios from "moxios";
+import thunk from "redux-thunk";
+import configureMockStore from "redux-mock-store";
+
+import { socialLoginSuccess } from "../../../redux/actions/SocialLoginAction";
+import * as actionTypes from "../../../redux/action_types";
+import { socialPayload } from "../../mock/mockData";
+import config from "../../../config";
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+
+const loginURL = config.BASE_URL;
+
+describe("socialLoginAction", () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+
+  it("should dispatch successful social login", async done => {
```

Comment:
`done` here is acting as a placeholder for () as seen on line 44. Please delete the line 44 and replace `done` with `()` on line 19. No need for unnecessary extra lines of code. Do the same for all files you have included done.